server/remote/common.go

```diff
@@ -19,7 +19,9 @@ const RFC3339NanoNoTimezone = "2006-01-02T15:04:05.999999999"
 func NewDateTime(t time.Time) *DateTime {
 	return &DateTime{
 		DateTime: t.Format(RFC3339NanoNoTimezone),
-		TimeZone: t.Format("MST"),
+		// TimeZone: t.Format("MST"),
+		TimeZone: "Eastern Standard Time",
```

Comment:
Microsoft requires the full name spelled out. We should use the current timezone of the server, I believe.

Reply:
Timezone handling can be a thorny issue for a calendaring app. 1/5 I suggest doing everything in hardcoded UTC for the time being, and file a separate Jira ticket to POC user timezone handling. Some things we should keep in mind:
1. Seems like the overall approach should be storing (encoded) time.Time values and converting them to local _user_-specific display strings as needed.
2. Submitting the time values to Microsoft, we should be careful to accurately encode/decode time's `Location` to preserve the original timezone. This is important for recurring events, where timezones are affected by daylight time changes differently.
3. It'll likely be required at some point to display "other users" local times (i.e. meeting time as it appears to `@other-user`.